src/transformers/modeling_utils.py

```diff
@@ -1380,8 +1380,9 @@ def _check_and_enable_flash_attn_2(
         """
         if not cls._supports_flash_attn_2:
             raise ValueError(
-                f"{cls.__name__} does not support Flash Attention 2.0 yet. Please open an issue on GitHub to "
-                "request support for this architecture: https://github.com/huggingface/transformers/issues/new"
+                f"{cls.__name__} does not support Flash Attention 2.0 yet. Please request to add support where"
+                " the model is hosted, on its model hub page: https://huggingface.co/[MODEL_REPO]/discussions/new"
```

Comment:
we might be able to get `self.config. _name_or_path?` for the path of the model

Reply:
Yep!